src/app/pages/i18n/i18n.component.scss

```diff
@@ -0,0 +1,112 @@
+@import '~@pxblue/colors/palette';
+.i18n-container {
+    height: 100vh;
+    width: 100%;
+}
+.sidenav {
+    max-width: 90%;
+    transition: width 200ms cubic-bezier(0.4, 0, 0.2, 1);
+    width: 350px;
+    .sidenav-content {
+        padding-top: 0;
+        overflow-x: hidden;
+    }
+    .toolbar-header {
+        background-color: map-get($pxb-blue, 500);
+        color: white;
+        min-height: 150px;
+    }
+    .logo-row {
+        height: 72px;
+        margin-left: 10px;
+        margin-bottom: 42px;
+        margin-right: 10px;
+        .logo {
+            font-size: 64px;
+            transform: rotate(42deg);
+        }
+    }
+    .nav-wrapper {
+        display: flex;
+        flex-direction: column;
+        flex: 1 1 0px;
+        overflow-y: auto;
+        overflow-x: hidden;
+        span {
+            margin: 0 16px;
+        }
+    }
+}
+
+.content {
+    background-color: map-get($pxb-white, 50);
+}
+
+.icon,
+.nav-wrapper mat-icon {
+    color: map-get($pxb-gray, 700);
+}
+.list {
+    color: map-get($pxb-gray, 500);
+}
+
+[matSubheader] {
+    white-space: nowrap;
+}
+
+mat-nav-list {
+    padding-top: 0px;
+    mat-checkbox {
+        display: inline-flex;
+    }
+}
+
+.select-language {
+    font-size: 1rem;
+    background-color: map-get($pxb-white, 50);
+    mat-form-field {
+        color: map-get($pxb-black, 500);
+    }
+}
+
+footer {
```

Comment:
agree that this should be centered in the main content (with drawer open or closed)

Reply:
I managed to get this centered by applying a margin-left to the snack-bar overlay container.   I originally wanted to provide a custom AppOverlayContainer that only overlaps the content found inside our DrawerLayout's content panel, but this idea didn't pan out.

I got this idea from this thread: https://github.com/angular/components/issues/7764

It worked well for the snackbar, but the snackbar is only 1 of 3 elements that use the overlay container.  The MatSelectItem and MatToolTip also use this Overlay, and when I overwrote the default AppOverlayContainer their positioning was a bit off.   Just used plain old css to fix this one.